### PR TITLE
fix: YIM Graph Overflow Issue

### PR DESCRIPTION
**Pull request by anshg1214**
This PR fixes the changes done in 3f96befd1afee70c0e9268e322ea62b144e5b58a. 

Issue Right Now:

https://github.com/user-attachments/assets/99b347e5-ec98-413c-8fa7-90637a7fa696

### CODE REVIEW

File: frontend/css/year-in-music.less [hunk 1]
```diff
@@ -568,7 +568,7 @@
       overflow: hidden;
       .graph {
         color: var(--selectedColor);
-        overflow: auto;
+        overflow: scroll;
```
Comment (MonkeyDo) [inline frontend/css/year-in-music.less:571]: This looks better IMO, and doesn't hide the tooltips:
```suggestion
        overflow: visible;
```

Before:
image


After:
image

**Review by MonkeyDo — CHANGES_REQUESTED** [review]
Thanks for finding the issue with this!
Suggestion to improve the result:

File: frontend/css/year-in-music.less (hunk 1 above)
Comment (anshg1214) [reply frontend/css/year-in-music.less:571]: If we set overflow to visible, it breaks on smaller screens 

image

Comment (MonkeyDo) [reply frontend/css/year-in-music.less:571]: Maybe`overflow-y: hidden;` then, I think that resolves both issues

Comment (anshg1214) [reply frontend/css/year-in-music.less:571]: Yep! It does the job

File: frontend/css/year-in-music.less [hunk 2]
```diff
@@ -568,7 +568,8 @@
...
```
Comment (MonkeyDo) [inline frontend/css/year-in-music.less:571]: I think this should allow  scrolling horizontally when needed (mobile sizes), while not showing the scrollbar when unnecessary:
```suggestion
        overflow-x: auto;
```

Comment (anshg1214) [reply frontend/css/year-in-music.less]: Done!

**Pull request merged**